genome_designer/genome_finish/graph_contig_placement.py

```diff
@@ -28,6 +29,52 @@
         ['exit_ref', 'enter_contig', 'exit_contig', 'enter_ref'])
 
 
+def get_genbank_features_with_type(genbank_path, feature_type=None):
+
+    chrom_intervals = {}
+    with open(genbank_path, 'r') as fh:
+        for seq_record in SeqIO.parse(fh, 'genbank'):
+            interval_list = []
+            for f in seq_record.features:
+                if feature_type and f.type == feature_type:
+                    interval_list.append((f, f.extract(seq_record.seq)))
+
+            chrom_intervals[seq_record.id] = interval_list
+
+    return chrom_intervals
+
+
+def write_me_features_multifasta(genbank_path, output_fasta_path,
+        also_inversions=False):
+    """Write mobile elements to file
+    """
+    me_features = get_genbank_features_with_type(
+            genbank_path, 'mobile_element')
+    me_sequences = {}
+    for chrom, feature_seq_tuples in me_features.items():
+        for feature, seq in feature_seq_tuples:
+            if 'mobile_element_type' not in feature.qualifiers:
+                continue
+            me_type = feature.qualifiers['mobile_element_type'][0]
+            me_type = me_type.replace(' ', '_')
+            if me_type not in me_sequences:
+                me_sequences[me_type] = seq
+
+    with open(output_fasta_path, 'w') as fh:
+        for me_type, seq in me_sequences.items():
+            fh.write('>ME_%s\n' % me_type)
+            fh.write(str(seq))
+            fh.write('\n')
+
+        if also_inversions:
+            for me_type, seq in me_sequences.items():
+                fh.write('>ME_REVERSE_%s\n' % me_type)
+                fh.write(str(seq)[::-1])
+                fh.write('\n')
+
+    fh.close()
```

Comment:
Don't need this when using `with` context wrapper. `fh` closed at the end of the `with`.